src/core/evaluator.js

```diff
@@ -765,7 +767,9 @@ var PartialEvaluator = (function PartialEvaluatorClosure() {
         }
         this.fontCache.put('id_' + fontID, fontCapability.promise);
       }
-      assert(fontID, 'The "fontID" must be defined.');
+      if (!fontID) {
```

Comment:
looks like fontID generated by internal logic. if we can tell that out logic produces non-empty id, we can keep assert.